### DISCUSSION TITLE
ThisIsWin11 1.1.0

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/ThisIsWin11#103] [Announcements]
# Release 1.1.0 of ThisIsWin11

- **Improved Packages module** and added next to the apps name a new apps ID attribute. This allows names to be assigned regardless of their installation ID in the Windows Package Manager manifest repository or Microsoft Store https://github.com/builtbybel/ThisIsWin11/pull/100 Thanks to @oleksis 
- Polished UI of Apps module aka PumpedApp
- Polished UI of Packages module
- **All modules now share the same UI style**
- **Added a more prominent placement of the _Restore default Windows 11 settings_ button** in system module aka OpenTweaks
- Updated Automation module
- Added some new Packages, e.g. NanaZip, [ElevenClock](https://github.com/martinet101/ElevenClock), [WSATools](https://github.com/Simizfo/WSATools)
- Minor improvements on Get started module
- Minor control fixes

![ThisIsWin11_yQK4Yk79Dk](https://user-images.githubusercontent.com/57478606/148675706-df77e077-8fae-40de-af54-d75bebc9150d.png)

_Please always download the TIW11.zip file as it contains the complete application package._

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/ThisIsWin11/releases/tag/1.1.0'>ThisIsWin11 1.1.0</a>.</em>

**Comment by DominikCr** [comment]
I'm not sure which script component broke Windows Hello (via fingerprint) for me. I tried to deselect -Privacy (disable) - "Windows Hello Biometrics "and ran the script again but to no avail. Any idea which other components are needed so the fingerprint sensor works again?

**Comment by Belim** [comment]
the only customization options to Windows Hello are led here

![ThisIsWin11_VA9qxnLXwq](https://user-images.githubusercontent.com/57478606/150634892-9b2f830b-a8c8-47c0-bed2-e3a871033db2.png)

it's just a switch in the registry. did you undo everything?

**Reply by DominikCr** [reply]
I did, but then taskbar looked too small so that the time or date was halfway below the screen and hardly visible. I ended up reinstalling win 11. 
btw, a former version of thisiswin (10 or maybe 11) had controls to get rid of cortana? I'm not seeing it in the latest build

**Comment by Trauni** [comment]
Short question: is this working on Windows 10 or is the old Windows 10 version still available in any link?
